smoketest/scripts/cli/test_protocols_bgp.py

```diff
@@ -131,7 +131,7 @@
         'remote_as'        : '200',
         'shutdown'         : '',
         'no_cap_nego'      : '',
-        'local_as'         : '300',
+        'system_as'         : '300',
```

Comment:
Please fix indention.

Reply:
I missed this. Thank you for seeing it. It should be fixed now. Sorry :(